src/loaders/compute_tools/microtrait/microtrait.py

```diff
@@ -107,6 +198,11 @@ def _run_microtrait(tool_safe_data_id: str, data_id: str, fna_file: Path, genome
 
     trait_counts_df.to_csv(os.path.join(genome_dir, loader_common_names.TRAIT_COUNTS_FILE), index=False)
 
+    heatmap_row, cells_meta, traits_meta = _process_trait_counts(trait_counts_df, data_id)
+    create_jsonl_files(genome_dir / MICROTRAIT_META, traits_meta)
```

Comment:
Is there a point in creating a meta file here?

Or do you need it because not all the column info is available in the rows / cells

Reply:
I want to create it so that the global parser can just read it and raise if it's there are differences between genomes. Otherwise, the global parser will need to read and build. I think by saving this will make the parser simpler.

Reply:
> Otherwise, the global parser will need to read and build. 

Not grokking what you mean by this

Reply:
I mean we need to save those trait meta info somewhere so that the global parser can read and build the `ColumnCategory` model. I would rather save it as a separate file than mixing it with cells data or meta file.

Reply:
:+1: